app/controllers/spina/journal/primer_theme/articles_controller.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module Spina
+  module Journal
+    module PrimerTheme
+      # User-facing controller for journal articles
+      class ArticlesController < ApplicationController
+        before_action :set_article, :set_issue, :set_breadcrumb, :set_metadata
+
+        def show
+          add_breadcrumb @article.title
+        end
+
+        private
+
+        def set_article
+          @article = Admin::Journal::Article.includes(affiliations: [:institution]).find(params[:id])
+          @article.view_context = view_context
```

Comment:
What was this for?

Reply:
I was trying to use Spina helpers but they weren't working, giving a view_context related error. So I added this by analogy with your controllers to try and fix it. It didn't work and I ended up doing what I was hoping to use the helpers for (images) via another method. So yeah this is vestigial.